mdbook/src/10-serial-port/README.md

```diff
@@ -0,0 +1,38 @@
+# Serial Port
+
+The closest thing to a universal I/O standard for modern day embedded boards is the "serial port". Pretty much every microcontroller has a way to talk serial on its pins, and pretty much every microcontroller board has these pins made easily accessible. The MB2 is no exception.
+
+In this chapter, we will describe what a serial port even is. We will then show you how to set up your computer with a "virtual serial port" using USB and use that virtual port with "terminal software" to talk to a serial port on the MB2.
+
+So what's this [serial port]? It's a place where two devices exchange data one bit at a time (*serially*) using one data line in each direction (*full-duplex*) plus a common ground. The serial port originated as "RS-232": see the history later in this chapter for details. However, the protocol spoken on the transmit and receive lines doesn't have an official name I'm aware of — it's just "serial" or maybe "async serial" or "UART serial".
```

Comment:
This is a bit inaccurate. Technically a USB port is also a serial port and there're serial interfaces like RS-485 which are not are not using one data line in each direction and typically not full-duplex. I2C and SPI are also quite typical serial busses. I think it'd make sense to go with the Wikipedia style of differentiation between parallel and serial communication and then go into why serial is beneficial (less connections => easier and cheaper and nowadays also fast).